libraries/AP_HAL_ChibiOS/hwdef/common/posix.h

```diff
@@ -79,14 +79,36 @@ typedef struct utimbuf
 #define MAX_NAME_LEN 13
 #endif
 
+enum
+  {
+    DT_UNKNOWN = 0,
+# define DT_UNKNOWN            DT_UNKNOWN
```

Comment:
I think the definition of definition is unnecessary.
I think it would be nice to use enum values in processing.

Reply:
This was copied from standard implementation, also It can be useful if there is a need for overriding the definition.